examples/proxy-server/src/main/java/example/armeria/proxy/ProxyService.java

```diff
@@ -0,0 +1,92 @@
+package example.armeria.proxy;
+
+import java.time.Duration;
+
+import com.linecorp.armeria.client.Endpoint;
+import com.linecorp.armeria.client.HttpClient;
+import com.linecorp.armeria.client.HttpClientBuilder;
+import com.linecorp.armeria.client.endpoint.EndpointGroup;
+import com.linecorp.armeria.client.endpoint.EndpointGroupRegistry;
+import com.linecorp.armeria.client.endpoint.EndpointSelectionStrategy;
+import com.linecorp.armeria.client.endpoint.StaticEndpointGroup;
+import com.linecorp.armeria.client.endpoint.healthcheck.HttpHealthCheckedEndpointGroup;
+import com.linecorp.armeria.client.endpoint.healthcheck.HttpHealthCheckedEndpointGroupBuilder;
+import com.linecorp.armeria.common.FilteredHttpResponse;
+import com.linecorp.armeria.common.HttpHeaderNames;
+import com.linecorp.armeria.common.HttpHeaders;
+import com.linecorp.armeria.common.HttpObject;
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.SessionProtocol;
+import com.linecorp.armeria.server.AbstractHttpService;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+public final class ProxyService extends AbstractHttpService {
+
+    private static final EndpointGroup animationGroup = new StaticEndpointGroup(
+            Endpoint.of("localhost", 8081),
+            Endpoint.of("localhost", 8082),
+            Endpoint.of("localhost", 8083));
+    // We used hardcoded backend address. But we can use service discovery mechanism to configure backends
```

Comment:
nit: How about moving this to the above of `animationGroup` and writing it as a Javadoc format?  We can make a link to DnsEndpointGroup.